diff -pur a/packages/build-deb-packages.sh b/packages/build-deb-packages.sh
--- a/packages/build-deb-packages.sh	2021-02-24 07:00:58.000000000 -0800
+++ b/packages/build-deb-packages.sh	2021-02-25 11:55:21.645540349 -0800
@@ -118,13 +118,14 @@ ex mv gdrcopy gdrcopy-${VERSION}
 ex tar czvf gdrcopy_${VERSION}.orig.tar.gz gdrcopy-${VERSION}
 
 ex cd ${tmpdir}/gdrcopy-${VERSION}
-debuild_params="--set-envvar=CUDA=${CUDA} --set-envvar=PKG_CONFIG_PATH=${PKG_CONFIG_PATH}"
+debuild_params="--set-envvar=CUDA=${CUDA} --set-envvar=PKG_CONFIG_PATH=${PKG_CONFIG_PATH} --set-envvar=KVER=${KVER}"
 if [ "${skip_dep_check}" -eq 1 ]; then
     debuild_params+=" --set-envvar=C_INCLUDE_PATH=${C_INCLUDE_PATH} --set-envvar=CPLUS_INCLUDE_PATH=${CPLUS_INCLUDE_PATH} --set-envvar=LIBRARY_PATH=${LIBRARY_PATH} --set-envvar=LD_LIBRARY_PATH=${LD_LIBRARY_PATH} -d"
     echo "Skip build dependency check. Use the environment variables instead ..."
 fi
 # --set-envvar needs to be placed before -us -uc
 debuild_params+=" -us -uc"
+yes | mk-build-deps -r -i
 ex debuild ${debuild_params}
 
 
diff -pur a/packages/debian/rules b/packages/debian/rules
--- a/packages/debian/rules	2021-02-24 07:00:58.000000000 -0800
+++ b/packages/debian/rules	2021-02-25 11:24:34.815473986 -0800
@@ -21,10 +21,10 @@
 # dh_make generated override targets
 # This is example for Cmake (See https://bugs.debian.org/641051 )
 override_dh_auto_build:
-	dh_auto_build -- CUDA=$(CUDA) lib exes
+	$(MAKE) config driver lib CUDA=$(CUDA)
 
 override_dh_shlibdeps:
 	dh_shlibdeps -Xcopybw -Xcopylat -Xsanity
 
 override_dh_auto_install:
-	dh_auto_install -- prefix=/usr libdir=/usr/lib/$(DEB_HOST_MULTIARCH)
+	$(MAKE) lib_install DESTDIR=debian/gdrcopy prefix=/usr libdir=/usr/lib/$(DEB_HOST_MULTIARCH)
